sources/procfs.go

```diff
@@ -296,6 +296,44 @@ func (s *lustreSource) generateMDSMetricTemplates() error {
 	return nil
 }
 
+func (s *lustreSource) generateClientMetricTemplates() error {
+	metricMap := map[string][]lustreHelpStruct{
+		"llite": {
+			{"blocksize", "blocksize_bytes", "Filesystem block size in bytes", s.gaugeMetric},
+			{"checksum_pages", "checksum_pages_enabled", "Returns '1' if data checksumming is enabled for the client", s.gaugeMetric},
+			{"default_easize", "default_ea_size_bytes", "Default Extended Attribute (EA) size in bytes", s.gaugeMetric},
+			{"filesfree", "inodes_free", "The number of inodes (objects) available", s.gaugeMetric},
+			{"filestotal", "inodes_maximum", "The maximum number of inodes (objects) the filesystem can hold", s.gaugeMetric},
+			{"kbytesavail", "kilobytes_available", "Number of kilobytes readily available in the pool", s.gaugeMetric},
+			{"kbytesfree", "kilobytes_free", "Number of kilobytes allocated to the pool", s.gaugeMetric},
+			{"kbytestotal", "kilobytes_capacity", "Capacity of the pool in kilobytes", s.gaugeMetric},
+			{"lazystatfs", "lazystatfs_enabled", "Returns '1' if lazystatfs (a non-blocking alternative to statfs) is enabled for the client", s.gaugeMetric},
+			{"max_easize", "max_ea_size_bytes", "Maximum Extended Attribute (EA) size in bytes", s.gaugeMetric},
+			{"max_read_ahead_mb", "max_read_ahead_megabytes", "Maximum number of megabytes to read ahead", s.gaugeMetric},
+			{"max_read_ahead_per_file_mb", "max_read_ahead_per_file_megabytes", "Maximum number of megabytes per file to read ahead", s.gaugeMetric},
+			{"max_read_ahead_whole_mb", "max_read_ahead_whole_megabytes", "Maximum file size in megabytes for a file to be read in its entirety", s.gaugeMetric},
```

Comment:
eh, yeah, I meant four. I edited the comment to say three, but clearly I can't count. Let's rephrase more clearly:

We should change any case of max to maximum instead. :)